pollux/lib/core/src/main/scala/io/iohk/atala/pollux/core/model/CredentialStatusList.scala

```diff
@@ -17,4 +17,36 @@ final case class CredentialStatusList(
     lastUsedIndex: Int,
     createdAt: Instant,
     updatedAt: Option[Instant]
-) {}
+)
+
+case class CredInStatusList(
```

Comment:
@shotexa, could you add the engineering docs for these case classes or draw any relational diagram?